colossalai/fx/passes/split_module.py

```diff
@@ -3,7 +3,7 @@
 from typing import Callable, List, Dict, Any, Optional
 from torch.fx._compatibility import compatibility
 from packaging import version
-from colossalai.fx.passes.utils import get_DAG
+from colossalai.fx.passes.utils import get_Topo
```

Comment:
The naming style is illegal

get_topology

Reply:
Done